### DISCUSSION TITLE
just guess.. file uploading time limited up to 5 mins ?

### DISCUSSION THREAD

**Discussion by magic-thomas** [livewire/livewire#5655] [1. Help] [answered]
Whenever  I upload big file which takes more than 5 minutes, it cause `401 Unauthorized` error. 
In dev tool , I see url `/livewire/upload-file?expires=1681466147&signature=..` . the expires value is corresponding to exact 5 minutes after I start uploading. 

I guess this 5 minutes limitation is hard-coded somewhere in source code. 

Somebody can confirm this ?

**Comment by joshhanley** [comment, marked as answer]
@magic-thomas yeah it’s validation applied to the temp upload for the side load. Publish the config and you can update it.

Hope this helps!
